runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java

```diff
@@ -411,4 +411,14 @@ default ShiftDepositMode bankDepositShiftClick()
 	{
 		return ShiftDepositMode.OFF;
 	}
+
+	@ConfigItem(
+		keyName = "swapEssenceMineTeleport",
+		name = "Essence Mine Teleport",
+		description = "Swaps Talk-To with Teleport for Wizard Distentor and Aubury."

```

Comment:
Why does this not swap teleport [for all of the essence mine teleport NPCs](https://oldschool.runescape.wiki/w/Rune_essence_mine#Teleportation_and_accessing_the_mine)?

Reply:
All others are impractical to use as a they are so far from a bank. Those are the only two anyone uses as per the second paragraph here: https://oldschool.runescape.wiki/w/Pure_essence#Obtaining_pure_essence

Reply:
I don't think that it matters that they are impractical to use, it doesn't make sense not to include all essence mine teleports in a config named `Essence Mine Teleport`.

Reply:
It makes sense to not have to maintain a massive list of NPC names statically defined in your codebase until the end of time, especially when virtually all of them are unused. Look at how it affects the style in MenuEntrySwapperPlugin.java. Feel free to close the request if you don't like it, I'll just keep using it locally myself.

Reply:
This could be solved by creating a constant `List` or `Set` with the names of these NPCs and checking `ESSENCE_TELEPORT_NPCS.contains(target)` instead of adding a new condition for each npc.